src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandomMobile.java

```diff
@@ -0,0 +1,126 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.contentpatterns.AdsFandomContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.dataprovider.ads.FandomAdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomMobileTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.ad.VideoFanTakeover;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsFandomObject;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+@Test(
+        groups = "AdsVuapFandomMobile"
+)
+public class TestAdsVuapFandomMobile extends AdsFandomMobileTestTemplate {
```

Comment:
`TestAdsVuapFandomDesktop` and `TestAdsVuapFandomDesktop` look very similar, would it make sense to write something more generic?